qwik-graphql-tailwind/src/components/repo-pulls/index.tsx

```diff
@@ -107,14 +86,15 @@ export default component$(({ activeTab, owner, name }: PullRequestsProps) => {
         {
           owner,
           name,
+          after,
+          before,
           first: DEFAULT_PAGE_SIZE,
           labels: dropdownStore.selectedLabel ? [dropdownStore.selectedLabel] : undefined,
-          orderBy: dropdownStore.selectedSort.split('^')[0],
-          direction: dropdownStore.selectedSort.split('^')[1],
+          orderBy: dropdownStore.selectedSort.split('^')[0] || '',
+          direction: dropdownStore.selectedSort.split('^')[1] || '',
```

Comment:
Maybe we can remove the `|| ' '` fallback case. We always have the default order set to "Newest"